File: src/main/java/util/PathBuilder.java
package ru.javarush.cryptoanalyzer.konovalov.util;

import java.io.File;
import java.nio.file.Path;

public class PathBuilder {
    private PathBuilder() {

    }

    public static String getTXT_FOLDER() {
        return TXT_FOLDER;
    }

    private static final String TXT_FOLDER = System.getProperty("user.dir") + File.separator + "text" + File.separator;

    public static Path getPath(String filename) {
        Path path = Path.of(filename);
        return path.isAbsolute() ? path : Path.of(getTXT_FOLDER() + filename);

    }
}

File: src/main/kotlin/util/PathBuilder.kt
package util

import org.jetbrains.annotations.NotNull
import java.io.File
import java.nio.file.Path

object PathBuilder {
    private val TXT_FOLDER: String = System.getProperty("user.dir")+ File.separator + "text" + File.separator

    fun getTXT_FOLDER() : String{
        return TXT_FOLDER
    }

    fun getPath(fileName: String) : Path {
        val path: Path = Path.of(fileName)
        return  if(path.isAbsolute) path else Path.of(getTXT_FOLDER()+fileName)

    }

}